docs/pages/api/inputs/rtp.md

```diff
@@ -1,6 +1,43 @@
+---
+title: RTP
+hide_table_of_contents: true
+---
+
 import Docs from "@site/pages/api/generated/renderer-RtpInputStream.md"
 
 # RTP
 An input type that allows streaming video and audio to the compositor over RTP.
 
+### Usage
+
+To use RTP Input you must register it first. You can do it by sending a request like this:
+
+<details>
+    <summary>Example request</summary>
+
+    ```http
+    POST: /api/input/:input_id/register
+    Content-Type: application/json
+    ```
+
+    ```js
+    {
+      "type": "rtp_stream",
+      "transport_protocol": "tcp_server",
+      "port": 9001,
+      "video": {
+        "decoder": "ffmpeg_h264"
+      },
+      "audio": {
+        "decoder": "opus"
+      },
+      "required": true,
```

Comment:
I think that we should emphasize in docs, that setting `required` to `true` for failable streams will stop the entire processing of the LiveCompositor once such input fails/stops sending data and avoid setting it to `true` in example requests.

Reply:
agree, but I don't think setting it to true is the issue here, specifying this field at all in the example is more of a problem